src/Generation/GapicClientGenerator.php

```diff
@@ -131,6 +131,7 @@ private function generateClass(): PhpClass
             ->withTrait($this->ctx->type(Type::fromName(\Google\ApiCore\GapicClientTrait::class)))
             ->withMember($this->serviceName())
             ->withMember($this->serviceAddress())
+            ->withMembers(Vector::new($this->hasServiceAddressTemplate() ? [$this->serviceAddressTemplate()] : []))
```

Comment:
Why do we have a `Vector` here?

Reply:
I did it because it may not exist, but looks like we can just supply `null` to `withMember` and achieve the same thing.
```suggestion
            ->withMember($this->hasServiceAddressTemplate() ? $this->serviceAddressTemplate() : null)
```

Reply:
Thanks for the suggestion!!